CHANGES.md

```diff
@@ -13,6 +13,8 @@
     for local libraries required during a module build.
 * base: add musl build. by @ericonr in
   https://github.com/cnpem/epics-in-docker/pull/35
+* base: add retools module. by @ericonr in
+  https://github.com/cnpem/epics-in-docker/pull/37
```

Comment:
This should be added in the Unreleased section after rebasing to v0.5.0. GitHub won't know how to rebase this correctly.